animareflection/ui
# Update `Tooltip` component

## Description

##### Task link: https://trello.com/c/DEXmihoF/236-tooltip-patches

1. Added `rounded` variant.
2. Added `arrow` prop to conditionally render the arrow component when desired.
3. Added `asChild` prop to TooltipTrigger.

## Test Steps

1. Verify the tooltip variant `rounded` works
2. Verify the `arrow` prop properly removes the arrow component on `arrow={false}`
3. Confirm the `asChild` prop will resolve the double nesting problem.
4. Check that all tests pass.
5. Verify the component renders in downstream apps.
